schema.ts

```diff
@@ -1135,6 +1134,103 @@ export interface IndexDefinition {
   popularityRankProperty?: PropertyIdentifier<string>;
 }
 
+/**
+ * Determines how permissions are handled for this object.
+ */
+export enum PermissionsBehavior {
+  /**
+   * The object will inherit permissions from its parent.
+   */
+  Inherit = 'Inherit',
+}
+
+export interface BaseParentDefinition {
+  /**
+   * The identity of the parent object.
+   */
+  parentIdentity: IdentityDefinition;
```

Comment:
Is this duplicative with `parentIdProperty`? That property would already include the reference to the other table.

Reply:
Yes, deleted